elearning-backend/app/Http/Controllers/API/UserController.php

```diff
@@ -9,13 +9,12 @@
 use App\Models\UserProfile;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\File;
-use Laravel\Sanctum\PersonalAccessToken;
 
 class UserController extends Controller
 {
     public function index(Request $request)
     {
-        $user = PersonalAccessToken::findToken($request->input('token'))->tokenable()->with('profile')->get()->first();
+        $user = User::where("id", auth()->user()->id)->with('profile')->get()->first();
```

Comment:
```suggestion
        $user = User::where("id", auth()->user()->id)->with('profile')->get();
        //or
        $user = auth()->user()->profile()
```
only use first if it has many relationship

Reply:
Changes has been made to file. Will also apply it to other files in separate bugfix branches.